packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -154,7 +187,10 @@ class ProductFullDetail extends Component {
                     </p>
                 </section>
                 <section className={classes.imageCarousel}>
-                    <Carousel images={product.media_gallery_entries} />
+                    <Carousel
+                        images={mediaGalleryEntries}
+                        key={mediaGalleryEntries}
```

Comment:
[The `key` attribute must be a string](https://reactjs.org/docs/lists-and-keys.html); `mediaGalleryEntries` appears to be an array. Is there another way to achieve the desired effect here?